src/WebServer/Http/HttpServer.cs

```diff
@@ -19,9 +20,11 @@ public class HttpServer : IDisposable
         private readonly StreamSocketListener _listener;
         private readonly SortedSet<RouteRegistration> _routes;
         private readonly ContentEncoderFactory _contentEncoderFactory;
+        private ILogger _log;
 
         public HttpServer(int serverPort)
         {
+            _log = LogManager.GetLogger<HttpServer>();

```

Comment:
I agree and I don't think ```IsErrorLevel``` methods are needed as we don't provide loglevel anyway.